### PR TITLE
CCI: Deploy RD2 to a managed org

### PR DESCRIPTION
**Pull request by force2b**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by force2b** [comment]
@rponti-sforg and @RadSFDeveloper Would you both mind reviewing & testing this PR when you get a chance? It fixes the `deploy_rd2_testing` flow and adds a new flow `deploy_rd2_testing_managed` specifically for use with beta/production orgs.

File: cumulusci.yml [hunk 1]
```diff
@@ -875,21 +875,42 @@ flows:
...
                 options:
                     class_name: RD2_DataMigration_BATCH
-            7:
-                task: execute_anon
-                # Execute the nightly job to update the RD records as needed
-                options:
-                    apex: >
-                        %%%NAMESPACE%%%RD2_OpportunityEvaluation_BATCH rdBatchJob = new %%%NAMESPACE%%%RD2_OpportunityEvaluation_BATCH();
-                        Database.executeBatch(rdBatchJob, rdBatchJob.batchSize);
+#            7:
```
Comment (RadSFDeveloper) [inline cumulusci.yml:878]: Since step 7 is not used, should we just remove it?

Comment (codexmarcel) [reply cumulusci.yml:878]: I am in favor of cleaning up that which is stale.

Comment (force2b) [reply cumulusci.yml:878]: 👍

**Comment by force2b** [comment]
@RadSFDeveloper @codexmarcel and @rponti-sforg I was thinking about the flow and task names a bit more. What would do you think about the following changes:

- Rename the `deploy_rd2_testing` and `deploy_rd2_testing_managed` flows to `enable_rd2` and `enable_rd2_managed` respectively. I think this is more inline with the `enable_crlp` task.
- Create a new task for `deploy_rd2_config_managed`. There's always confusion on how to use the CCI task to deploy the RD2 unpackaged config into a beta org (adding the `-o managed false` parameter). Making a new task that has that option built in might make it easier for everyone (me too).

**Comment by jesslopez-sf** [comment]
Thank you for creating the flow for managed orgs, @force2b !

**Comment by RadSFDeveloper** [comment]
> @RadSFDeveloper @codexmarcel and @rponti-sforg I was thinking about the flow and task names a bit more. What would do you think about the following changes:
> 
> * Rename the `deploy_rd2_testing` and `deploy_rd2_testing_managed` flows to `enable_rd2` and `enable_rd2_managed` respectively. I think this is more inline with the `enable_crlp` task.
> * Create a new task for `deploy_rd2_config_managed`. There's always confusion on how to use the CCI task to deploy the RD2 unpackaged config into a beta org (adding the `-o managed false` parameter). Making a new task that has that option built in might make it easier for everyone (me too).

Sounds good @force2b !

**Comment by rponti-sforg** [comment]
> > @RadSFDeveloper @codexmarcel and @rponti-sforg I was thinking about the flow and task names a bit more. What would do you think about the following changes:
> > 
> > * Rename the `deploy_rd2_testing` and `deploy_rd2_testing_managed` flows to `enable_rd2` and `enable_rd2_managed` respectively. I think this is more inline with the `enable_crlp` task.
> > * Create a new task for `deploy_rd2_config_managed`. There's always confusion on how to use the CCI task to deploy the RD2 unpackaged config into a beta org (adding the `-o managed false` parameter). Making a new task that has that option built in might make it easier for everyone (me too).
> 
> Sounds good @force2b !

I agree 100%.  Regarding the second bullet, that is exponentially helpful.

**Pull request merged**